scripts/gitlab/validate-chainspecs.sh

```diff
@@ -15,6 +15,15 @@ done
 for spec in ethcore/res/ethereum/*.json; do
     if ! ./target/release/chainspec "$spec"; then ERR=1; fi
 done
+
+echo "________Mainnet contains Istanbul EIPs________"
+for eip in $(grep --only-matching "eip.*Transition" ethcore/res/ethereum/istanbul_test.json); do
```

Comment:
how deal with `re-enable transitions`?

Reply:
Add them to the chain spec?
https://github.com/paritytech/parity-ethereum/pull/11228#discussion_r341586118

Reply:
yeah, I guess that is easiest solution even though they are redundant in this case.